docs/_data/swagger.yml

```diff
@@ -736,6 +736,8 @@ paths:
                 ```?page={page-number}``` Default page number is 0
 
                 ```?pagesize={pagesize}``` Default pagesize is 10
+
+                ```?filter={filter}``` Filter connections with type or sub_type
```

Comment:
Did you manually edited this file?

Reply:
https://github.com/meshery/meshery/pull/9729/files#diff-93e0222a2d4ff5b9d401573f795ea74715141d16731ac18c08af5d109128b1d8R191

Update this comment ad then run `make swagger-build`.

Reply:
Will you also add an example of the filter param, stating how it can be used?